src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  function getSum(a) {
+    return function(b) {
```

Comment:
You can add 0 as a default value. So if no argument is passed, it would still work.